.github/workflows/spider-check.yaml

```diff
@@ -20,9 +20,18 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.repository_owner == 'jquery' }} # skip on forks
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout hydra-link-checker
+        uses: actions/checkout@v3
         with:
           repository: jquery/hydra-link-checker
           ref: v2.0.0
+          path: hydra
+
+      # Checkout the API repo as well to provide the config for hydra-link-checker
+      - name: Checkout API repo
+        uses: actions/checkout@v3
+        with:
+          path: api
```

Comment:
Is there a better way to provide a single file to another step?

Reply:
Not that I know of. LGTM.